repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/step/repair/NopolRepair.java

```diff
@@ -27,7 +27,7 @@
 import java.util.*;
 import java.util.concurrent.*;
 
-import static fr.inria.spirals.repairnator.process.git.GitHelper.*;
+import static fr.inria.spirals.repairnator.process.files.FileHelper.*;
```

Comment:
I personnally prefer if we import FileHelper and call the static method by using `FileHelper.deleteFile()` or something like that: it's easier to identify where the methods are coming from in case of debugging.